src/main/java/com/mvplevel1/exception/QualifyLoan.java

```diff
@@ -0,0 +1,9 @@
+package com.mvplevel1.exception;
+
+public class QualifyLoan extends RuntimeException{
```

Comment:
For all custom exceptions there is a special naming convention - names should ends with "...Exception" word. In this case QualifyLoanException will be more suitable